validator/remote/src/main/java/tech/pegasys/teku/validator/remote/apiclient/OkHttpValidatorRestApiClient.java

```diff
@@ -127,8 +135,21 @@ public Optional<GetBlockHeaderResponse> getBlockHeader(final String blockId) {
         createHandler(GetBlockHeaderResponse.class));
   }
 
+  /**
+   * Tries <a
+   * href="https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/postStateValidators">POST
+   * Get validators from state</a> and falls back to <a
+   * href="https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/getStateValidators">GET
+   * Get validators from state</a> if POST method is not allowed
+   */
   @Override
   public Optional<List<ValidatorResponse>> getValidators(final List<String> validatorIds) {
+    try {
+      return postValidators(validatorIds);
```

Comment:
this is kind of the wrong place...
Basically getValidators has to split on id, and we don't have to with the post. It'd be better if we're determining to use post and calling for ALL keys in 1 call, rather than this approach, but it could potentially be another PR...

Reply:
Great catch, on some test node with 5000 validators we could have 100 bad post calls and 100 good gets when it's not supported and 100 post calls instead of 1 when it's supported.

Reply:
Got it. Working on the changes. :)